analysisservice/src/main/java/nu/yona/server/AnalysisServiceApplication.java

```diff
@@ -51,6 +51,6 @@ public Docket yonaApi()
 	@Bean
 	public CacheManager cacheManager()
 	{
-		return new ConcurrentMapCacheManager("goalConflictMessages");
+		return new ConcurrentMapCacheManager("goalConflictMessages", "usersAnonymized");
```

Comment:
Agreed. However, I expect the overhead to be negligible as long as we don't populate the cache.